pyrsa/rdm/rdms.py

```diff
@@ -463,9 +464,58 @@ def rank_transform(rdms, method='average'):
                                 for i in range(rdms.n_rdm)])
     rdms_new = RDMs(dissimilarities,
                     dissimilarity_measure=rdms.dissimilarity_measure,
-                    descriptors=rdms.descriptors,
-                    rdm_descriptors=rdms.rdm_descriptors,
-                    pattern_descriptors=rdms.pattern_descriptors)
+                    descriptors=deepcopy(rdms.descriptors),
+                    rdm_descriptors=deepcopy(rdms.rdm_descriptors),
+                    pattern_descriptors=deepcopy(rdms.pattern_descriptors))
+    return rdms_new
+
+
+def sqrt_transform(rdms):
```

Comment:
I'd argue for putting these in a separate "transforms" module but that's subjective

Reply:
I am fine with this at least, so I did it